tests/nnstreamer_datarepo/unittest_datareposrc.cc

```diff
@@ -77,15 +76,15 @@ create_sparse_tensors_test_file ()
   gchar *file_path = get_file_path (filename);
   gchar *json_path = get_file_path (json);
 
-  const gchar *str_pipeline = g_strdup_printf (
+  gchar *str_pipeline = g_strdup_printf (
       "datareposrc location=%s json=%s start-sample-index=0 stop-sample-index=9 ! "
       "tensor_sparse_enc ! other/tensors,format=sparse,framerate=0/1 ! "
       "datareposink location=sparse.data json=sparse.json",
       file_path, json_path);
 
   GstElement *pipeline = gst_parse_launch (str_pipeline, NULL);
   ASSERT_NE (pipeline, nullptr);
-
+  g_free (str_pipeline);
```

Comment:
recommend moving this before assert (line 86) to avoid svace issue;

Reply:
Thank you.